pkg/gvk/gvk.go

```diff
@@ -239,12 +267,20 @@ func resourceExist(resource metav1.APIResource, resources []metav1.APIResource)
 	return false
 }
 
-func compareResources(src []*metav1.APIResourceList, dst []*metav1.APIResourceList) []*metav1.APIResourceList {
+func (r *Compare) compareResources(src []*metav1.APIResourceList, dst []*metav1.APIResourceList) []*metav1.APIResourceList {
 	missingResources := []*metav1.APIResourceList{}
+	sortResources(src)
 	for _, srcList := range src {
 		missing := []metav1.APIResource{}
 		for _, resource := range srcList.APIResources {
-			if !resourceExist(resource, fingResourceList(srcList.GroupVersion, dst)) {
+			if cohabitator, found := r.CohabitatingResources[resource.Name]; found {
+				if cohabitator.seen {
+					// TODO: need to add log entry
```

Comment:
@alaypatel07 you might be interested in learning about the controller `logging` package https://github.com/konveyor/mig-controller/blob/cac36386f9de729b65c452a453825254a6b9fe12/docs/devguide.md#logging

Reply:
In mig-controller the logging philosophy is essentially "less is more". We rely heavily on CR conditions to convey information as needed. 

mig-controller only writes logs when:

1) an error occurs
2) it enters or exits stages in an active migration
3) controller or watches start/stop

Reply:
In this case I would argue that the resulting MigPlan condition conveying GVK diff information would be sufficient, no log message required. Open to counter-argument.

Reply:
ack, not logging here makes sense in this case. Upstream Velero logs here, but I was not sure if we want our controller to log. I'll remove the TODO.